code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -1111,13 +1127,6 @@
 		new /obj/item/stack/cable_coil(loc, 3)
 	qdel(src)
 
-/obj/machinery/alarm/AltClick(mob/user)
-	if(Adjacent(user) && allowed(user) && !wires.is_cut(WIRE_IDSCAN))
-		locked = !locked
-		to_chat(user, "<span class='notice'>You [locked ? "lock" : "unlock"] the Air Alarm interface.</span>")
-	else
-		to_chat(user, "<span class='warning'>Access denied.</span>")
-
```

Comment:
This shouldn't be removed as it got added by a recent PR

Reply:
Rebased branch based on an updated one with the same changes (I think)